tests/scripts/delete_all_indexes.py

```diff
@@ -3,22 +3,22 @@
 import marqo
 
 
-def cleanup_documents_from_all_indices():
+def delete_all_test_indices():
     local_marqo_settings = {
         "url": os.environ.get("MARQO_URL", 'http://localhost:8882'),
     }
+    suffix = os.environ.get("MARQO_INDEX_SUFFIX", None)
     api_key = os.environ.get("MARQO_API_KEY", None)
     if api_key:
         local_marqo_settings["api_key"] = api_key
     client = marqo.Client(**local_marqo_settings)
     indexes = client.get_indexes()
     for index in indexes['results']:
-        if client.config.is_marqo_cloud:
-            if index.get_status()["index_status"] == marqo.enums.IndexStatus.CREATED:
-                index.delete()
-        else:
-            index.delete()
+        if index.index_name.startswith('test-index'):
+            if suffix is not None and suffix in index.index_name.split('-'):
+                if index.get_status()["index_status"] == marqo.enums.IndexStatus.CREATED:
+                    index.delete()
```

Comment:
Question: so we don't delete open source indexes because this is delegated to the marqo_test logic?

If so, then we can rename the file and function to be more specific: `delete_all_cloud_test_indexes.py` for example